src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -4,11 +4,25 @@ import { InvalidArgumentError } from '../../../common/error/errors';
 import { OrderActionCreator, OrderRequestBody } from '../../../order';
 import { OrderFinalizationNotRequiredError } from '../../../order/errors';
 import { PaymentArgumentInvalidError, PaymentMethodInvalidError } from '../../errors';
+import { PaymentStrategyType } from '../../index';
 import PaymentActionCreator from '../../payment-action-creator';
 import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
 import PaymentStrategy from '../payment-strategy';
 
-import { ApproveDataOptions, ButtonsOptions, PaypalCommerceCreditCardPaymentInitializeOptions, PaypalCommerceFundingKeyResolver, PaypalCommerceInitializationData, PaypalCommercePaymentInitializeOptions, PaypalCommercePaymentProcessor, PaypalCommerceScriptParams } from './index';
+import { ApproveDataOptions,
+    ButtonsOptions,
+    PaypalCommerceCreditCardPaymentInitializeOptions,
+    PaypalCommerceFundingKeyResolver,
+    PaypalCommerceInitializationData,
+    PaypalCommercePaymentInitializeOptions,
+    PaypalCommercePaymentProcessor,
+    PaypalCommerceRequestSender,
+    PaypalCommerceScriptParams } from './index';
+
+const ORDER_STATUS_APPROVED = 'approved';
+const ORDER_STATUS_CREATED = 'created';
```

Comment:
I see that `OrderStatus` interface has a `string` type for the `status` property, what if we make the `status` type more specific? I think it will be more clear and helps with understandability 

Current interface:
```
export interface OrderStatus {
    status: string;
}
```

New example:
```
export interface OrderStatus {
    status: 'approved' | 'created'; // don't foget to add all possible types here
}
```

Reply:
fixed